pynautobot/core/response.py

```diff
@@ -280,8 +280,9 @@ def _endpoint_from_url(self, url):
             url_path = url_path[len(extra_path) :]
         split_url_path = url_path.split("/")
         if split_url_path[2] == "plugins":
-            # Skip plugins from the path
-            app, name = split_url_path[3:5]
+            # Keep plugins in app path
+            app = "/".join(split_url_path[2:4])
+            name = split_url_path[4]
```

Comment:
```suggestion
            # Keep plugins in app path
            app, name = split_url_path[3:5]
            return getattr(pynautobot.core.app.PluginsApp(self.api, app), name)
```

Reply:
I'm not sure if this works, but this is what I would expect

Reply:
Sadly that does not work. AFAICT `PluginsApp` is written somehow strangle as it does not extend original `App` and just adds `plugins/` to the url, but instead it is completely new object, so this way you never get to `App.__getattr__`. And it has different `__init__`. The result is your suggestion cannot work as `PluginsApp` does no accept `app`. If I try to pass the whole string there as `app/name` in place of `name`, then `Endpoint.url` does not get set as string so it crashes in `Endpoint.normalize_url` function.
So to fix #13 it either requires my easy-but-ugly fix or refactor `PluginsApp`...